tests/test_dojo.py

```diff
@@ -0,0 +1,264 @@
+"""Unit tests for the application"""
+
+import unittest
+import io
+import sys
+import colorful
+from prettytable import PrettyTable
+from src.dojo import Dojo
+
+
+class TestSpaceAllocator(unittest.TestCase):
```

Comment:
Just like in your previous files, your file name should give me an idea of what classes I should expect inside the file.
e.g In test_dojo.py , I expected to find a test_dojo.py class.